_config.yml

```diff
@@ -198,6 +198,24 @@ flink_kubernetes_operator_releases:
         asc_url: "https://downloads.apache.org/flink/flink-kubernetes-operator-0.1.0/flink-kubernetes-operator-0.1.0-helm.tgz.asc"
         sha512_url: "https://downloads.apache.org/flink/flink-kubernetes-operator-0.1.0/flink-kubernetes-operator-0.1.0-helm.tgz.sha512"
 
+flink_table_store_releases:
+  -
+    version_short: "0.1"
+    source_release:
+      name: "Apache Flink Table Store 0.1.0"
+      id: "010-table-store-download-source"
+      flink_version: "1.15.0"
+      url: "https://www.apache.org/dyn/closer.lua/flink/flink-table-store-0.1.0/flink-table-store-0.1.0-src.tgz"
+      asc_url: "https://downloads.apache.org/flink/flink-table-store-0.1.0/flink-table-store-0.1.0-src.tgz.asc"
+      sha512_url: "https://downloads.apache.org/flink/flink-table-store-0.1.0/flink-table-store-0.1.0-src.tgz.sha512"
```

Comment:
I see those links are reporting `404 Not founds`.  Does it means we leave those links here unless we released those packages into apache maven repository ( I see we are still voting for the release in the  mail list ).

Reply:
This PR will be merged when vote has been passed. Then the links will be aviliable.